recipe/meta.yaml

```diff
@@ -8,41 +8,59 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 1e4c4a229044953c5740c31f1f9b38df01420f595d11c22c9d106845d6b5e1d3
-
+  patches:
+    - patches/0001-replace-relative-imports-with-module.patch
+    - patches/0002-fix-matplotlib-axes-subplot.patch
+    - patches/0003-fix-another-relative-import.patch
+    - patches/0004-fix-more-matplotlib-axes-subplot.patch
+    - patches/0005-remove-upper-bounds-from-setup-py.patch
+    - patches/0006-remove-upper-bounds-from-pyproject-toml.patch
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  # missing h3-py, python-igraph for 3.12
+  skip: True  # [py<38 or py>311 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - patch  # [not win]
+    - m2-patch  # [win]
   host:
-    - python >=3.6
+    - python
     - pip
-    - poetry
+    - poetry-core
   run:
```

Comment:
The list of runtime dependencies is different https://github.com/scikit-mobility/scikit-mobility/blob/1.3.1/pyproject.toml#L33-L44

Reply:
They don't do a great job listing dependencies. I've checked all of the "extra" ones, they are used by source code.

Reply:
Ah, that explains a lot. Thanks!